### DISCUSSION TITLE
Startup

### DISCUSSION THREAD

**Discussion by fanhousanbu** [go-to-cloud/go-to-cloud#1] [General]
## 项目启动条件

1. 项目定位
2. 项目边界
3. 成功标志

---

> 请各位根据启动条件提出自己的看法，我们最终汇总成一份大家共同认可的目标

**Comment by fanhousanbu** [comment]
### 项目定位

服务于`15`人左右规模的团队从创建代码仓库至发布上线为止的CI/CD一站式工具

### 项目边界

专注于代码的编译、产物的部署（回滚）；支持以插件方式扩大项目边界能力

### 成功标志

stars > 1k

**Reply by fanhousanbu** [reply]
> ### 项目定位
> 服务于`15`人左右规模的团队从创建代码仓库至发布上线为止的CI/CD一站式工具
> 
> ### 项目边界
> 专注于代码的编译、产物的部署（回滚）；支持以插件方式扩大项目边界能力
> 
> ### 成功标志
> stars > 1k

不允许以刷单方式完成

**Comment by sampsonye** [comment]
### 产品定位
1. 隔离开发&运维，保持各自的专注度（开发专注代码，运维专注基础设施）
2. 最快上手的面向云原生的持续集成工具
3. 服务中小型公司或团队

### 产品边界
1. 构建、部署（回滚）、质量能力的全覆盖
2. 服务发布&管理、产物发布&管理

### 成功标志
1. 有5家以上客户在真正使用我们的产品
